minimum-constraints.txt

```diff
@@ -311,7 +312,8 @@ jupyterlab-widgets==1.0.2; python_version == '3.6'
 jupyterlab-widgets==3.0.3; python_version >= '3.7'
 
 jupyter-server>=1.24.0; python_version == '3.7'
```

Comment:
It is good that you changed >= to == in the other entry for jupyter-server. That should also be done here.

Reply:
Hmm.  I show that one as changed.  In any case change.

Reply:
The change I see is on the line below the one I commented on.

Reply:
My local file shows that changed.  Should be in latest push this morning.

Reply:
It is not yet in the latest PR as of now.